SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -443,7 +443,7 @@ TSharedPtr<FUnrealType> CreateUnrealTypeInfo(UStruct* Type, uint32 ParentChecksu
 			PropertyInfo->HandoverData = MakeShared<FUnrealHandoverData>();
 			PropertyInfo->HandoverData->Handle = HandoverDataHandle++;
 		}
-		return true;
+		return false;
```

Comment:
big fix

Reply:
Does this mean that another layer of nested structs won't be handed over? I mean, if we had
```
struct S1
{
	UPROPERTY()
	int S1Value;
};

struct S2
{
	UPROPERTY()
	S1 Other;

	UPROPERTY()
	int S2Value;
};

class MyActor
{
	UPROPERTY(Handover)
	S2 HandoverProperty;
};
```
then `S1::S1Value` won't be handed over, but `S2::S2Value` would? We may want to add a known issue or something since this diverges from Native.

Reply:
Could you share what the schema looks like in the above instance?

Reply:
```
component HandoverReplicationTestCubeHandover {
	id = 12087;
	int32 handovertestproperty = 1;
	bytes handoverteststruct = 2;
	int32 teststage = 3;
}
```
just bytes
no idea how Unreal just works with this, but apprently it does